snapcraft/cli/lifecycle.py

```diff
@@ -202,6 +202,23 @@ def _run_pack(snap_command: List[Union[str, pathlib.Path]]) -> str:
 def _pack(
     directory: str, *, compression: Optional[str] = None, output: Optional[str]
 ) -> None:
+    output_file = None
+    output_dir = None
+
+    # Output may be:
```

Comment:
This should be in the doctstring for the method instead of a code comment

Reply:
Done!